tests/unit/helpers.spec.ts

```diff
@@ -231,7 +231,8 @@ describe('IResponseFormat validation', () => {
     const siteId = 123;
     const siteName = 'test site';
     const dataSource = 'FMI';
-    const depth = 10;
+    const sampleDepth = '10';
```

Comment:
Tällä tuskin on väliä, mutta tuossa ylemmässä asetetaan datan lähteeksi FMI, mutta FMI-datalla ei ole syvyystietoja. Pitäsköhän dataSource vaihtaa, niin vastais enemmän oikeeta tilannetta?

Reply:
No näitä käytetään sekaisin noissa vesla ja fmi resulteissa, eipä sillä väliä mikä stringi siellä on